docs/pages/config.mdx

```diff
@@ -12,18 +12,19 @@ The config is used to define:
   If you want greater control over your systems (to change their public access or their name), you can use the `systems` object in the config.
 - The [modules](/world/modules) that will be installed in the `World`.
 
-The is an example of a `World` config:
+This is an example of a `World` config:
 
 ```tsx
 import { defineWorld } from "@latticexyz/world";
+import { resolveTableId } from "@latticexyz/world/internal";
```

Comment:
We should not document things that live in `/internal`, the idea with those exports is that they're only used internally. This makes me wonder though whether we should make this util an external export since it's needed for this use case @holic?

Reply:
I intentionally made it internal because I am not happy with this API and want to replace it without it being considered a breaking change